submarine-server/server-api/src/main/java/org/apache/submarine/server/api/environment/Environment.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.submarine.server.api.environment;
+
+import org.apache.submarine.server.api.spec.EnvironmentSpec;
+
+public class Environment {
+
+  /**
+   * ID of the environment
+   */
```

Comment:
Why we need an ID here? is a unique name sufficient?

Reply:
Name is sufficient. In general, Having auto increment identifier for table is a good practice to do any CRUD operations in future. Just used that in Environment object as well.

Reply:
Make sense. However, I suggest using a string for ID which is more flexible and consistent with other best practices (like K8s). Incremental ID could cause clashing for HA cases.

Reply:
Taken care.